### DISCUSSION TITLE
Code Style

### DISCUSSION THREAD

**Discussion by AnakinRaW** [AlamoEngine-Tools/PetroglyphTools#327] [General]
I noticed that current code stlyle for this repo wants explicit types and not using `var`. 

Is this anything we care about @gruenwaldlk? Personally i favour `var`. 

Same for private instance fields. In the past the m_varName style was used. While I see somewhat an advantage to it in theory, i never really experienced it in practice. Also i rarely see this style in the .NET source code, which is what I'd call the de-facto standard as coding style. 

The other styling things fit my preferences :)

**Comment by gruenwaldlk** [comment]
As far as `var` usage goes I personally don't particularly care either way, from experience working with code bases written by other teams though, explicit types can make maintenance easier down the road. I do like consistency in a code base though, so `var` should be the default, unless there is immediate benefit to having the type spelled out.

The `m_variableName` style I've encountered in the industry as a common style (just like `s_staticPrivateVariable`, ...) but I'd go with the .NET format over that, just for consistency's sake.
